apps/modernization-ui/src/apps/page-builder/components/EditPageSidebar/EditPageSidebar.scss

```diff
@@ -27,7 +21,7 @@
         display: flex;
         align-items: center;
 
-        .item-label {
+        .edit-page-sidebar__button--label {
             margin-left: 16px;
```

Comment:
should we use rem for everything?  I honestly have never used rem before so we should probably discuss at some point.

Reply:
yes rems always